pom.xml

```diff
@@ -35,6 +43,17 @@
       </properties>
     <build>
     <plugins>
+          <plugin>
+            <groupId>com.github.inodb.maven-external-version</groupId>
```

Comment:
Should this (eventually) move from this privately-maintained repo to a cBioPortal community effort?

Reply:
@fedde-s i can fork it to the cBioPortal repo to make it look more official before merging

Reply:
Ah, I see that it's a fork of an existing project. If the changes you made get accepted upstream, it might get even more people to look at it and maintain it. Do what you feel is best! 🙂

Reply:
@fedde-s yeah it's unfortunately not actively maintained 😢. Im not sure why, seems like a lot of people still use it and I haven't found a good alternative. I've forked it to our org and made a single commit that describes the changes I made better: https://github.com/cBioPortal/maven-external-version/commit/67e9fc4ebb17333b119ab10e1b521fe8391b7ff2. It's just for allowing one to pull the dependency from jitpack (the plugin is not published on maven central)